Documentation/tuning.md

```diff
@@ -18,18 +18,22 @@ This timeout is how long a follower node will go without hearing a heartbeat bef
 By default, etcd uses a `1000ms` election timeout.
 
 Adjusting these values is a trade off.
-Lowering the heartbeat interval will cause individual commands to be committed faster but it will lower the overall throughput of etcd.
-If your etcd instances have low utilization then lowering the heartbeat interval can improve your command response time.
+The value of heartbeat interval is recommended to be around the maximum of average round-trip time(RTT) between members, normally in the range [0.5\*RTT, 1.5\*RTT].
```

Comment:
either way, are you trying to say:

in the range of 0.5-1.5x the round trip time? If so, can we state it like that?